### PR TITLE
Add contribution context and template

### LINKED ISSUE #5: Write contribution guidelines
They could be useful

### PR DESCRIPTION
**Pull request by bradrf**
This adds some more context around contribution along with a pull request template to help folks provide better information when making suggestions.

Closes #5

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -4,16 +4,42 @@ Please note that this project is released with a [Contributor Code of Conduct](c
...
+- Have a decently active community and engaged ownership.
+
+## Pull Request Style
+
+#### Ground Rules
+
+- No duplicates.
+- Individual pull request for each suggestion.
+- The repository is older than 90 days (where applicable).
+- The repository has more than 10 stars (where applicable).
```
Comment (aloisdg) [inline contributing.md:27]: Could we request something like 50 stars? 30? less? more? The main goal of awesome list is to show what the community find awesome. I think it is quite fair to request some "approvals".

For non repository, we could ask for reddit/HN/lobsters link. 

WDYT?

File: contributing.md [hunk 2]
```diff
@@ -4,16 +4,42 @@ Please note that this project is released with a [Contributor Code of Conduct](c
...
+- The item is added at the bottom of the relevant category.
+- Use the following format: `[ITEM_NAME](LINK) - DESCRIPTION.`
+  - Description starts with a capital and ends with a full stop (period).
+    - Try to keep it short and concise.
+  - No trailing whitespace.
+  - Check your spelling and grammar.
+
+### Other Pull Requests
+
+Pull requests that introduce new categories or improve the readme are always welcome! In that case feel free to disregard the given structure above.
```
Comment (aloisdg) [inline contributing.md:41]: :+1:

**Review by aloisdg — COMMENTED** [review]
Great :smile:

File: contributing.md (hunk 1 above)
Comment (aloisdg) [reply contributing.md:27]: cc @Knusper

Comment (bradrf) [reply contributing.md:27]: Yep, I think it'd be fine to start out higher and drop it if we find it too restrictive. I'll bump it to 50.

That's an excellent idea re:reddit/hn/lobsters link for corroboration against non-source items. I'll definitely add that, too.

**Pull request merged**